src/dialog/DefautIconSettings.ts

```diff
@@ -0,0 +1,32 @@
+import { getGame } from "../system/util";
+import Options = FormApplication.Options;
+
+export class DefaultIconSettings extends FormApplication {
+  constructor(object: any, options?: Partial<Options>) {
+    super(object, options);
+  }
+
+  static get defaultOptions(): Options {
+    return mergeObject(super.defaultOptions, {
+      id: "default-icons-settings-dialog",
+      title: "Default Item Icon Settings",
+      template: "./systems/rqg/dialog/defaultIconSettings.hbs",
+      classes: ["form", "default-icons"],
+      width: 500,
+      closeOnSubmit: false,
+      submitOnChange: true,
+    });
+  }
+
```

Comment:
Adding this will update the img icon immediately when selecting an image file. And then the OK button isn't really neccesary (since it doesn't actually do anything).
```suggestion
  protected _onSelectFile(selection: string, filePicker: FilePicker): void {
    super._onSelectFile(selection, filePicker);
    this.submit();
  }
```
At some point one might want to have a save / cancel button there, but that sounds like unneccesary work for now.